watertap/unit_models/electroNP_ZO.py

```diff
@@ -83,6 +84,10 @@ def build(self):
 
         units_meta = self.config.property_package.get_metadata().get_derived_units
 
+        add_object_reference(self, "proerties_in", self.mixed_state)
```

Comment:
We should probably have test for the refs. Note the typo here which wouldn't be caught since we don't test it.